kart/json_diff_writers.py

```diff
@@ -98,6 +107,11 @@ def default(self, obj):
                 result["feature"] = self.filtered_ds_feature_deltas_as_json(
                     ds_path, ds_diff
                 )
+            if "tiles" in ds_diff:
+                result["tiles"] = [
+                    self._postprocess_tiles_delta(key, value)
+                    for key, value in ds_diff["tiles"].items()
+                ]
```

Comment:
So until now `tiles` here is a dict/object:

```
{
    "mytilename1.laz": {"-": {...}, "+": {...}},
    ...
}
```

and this thing turns it into a list

```
[
    {"-": {"name": "mytilename1.laz", ...}, "+": {"name": "mytilename1.laz", ...}},
    ...
]
```

Could you add a comment explaining what's going on here? It's not very clear to me